packages/pixelboy/apps/journal/checkin/checkin.coffee

```diff
@@ -18,12 +18,14 @@ class PAA.PixelBoy.Apps.Journal.CheckIn extends AM.Component
   events: ->
     super.concat
       'submit .check-in-form': @onSubmitCheckInForm
+      'input .external-url': @getExternalImgUrl
```

Comment:
Handlers should have `on<EventType><Trigger>` format. In your example `onInputExternalUrl`. If you wanted the image part the class should also be `.external-image-url`. And `Image`, not `Img`, we (almost) never abbreviate. From [General Guidelines](https://github.com/peerlibrary/peerlibrary/wiki/Code-Style#general-guidelines):
> Be verbose, always spell out all names. You will sometimes see widely used abbreviations like min, max and id that have no chance to be ambiguous. That is OK. Use itemCount instead of numItems.
In your defense, you could argue that img is widely used and not ambiguous and you'd be right. So Image for consistency sake (and only adds 2 extra characters).